v2/pkg/protocols/common/expressions/expressions_test.go

```diff
@@ -12,6 +12,7 @@ func TestEvaluate(t *testing.T) {
 		expected string
 		extra    map[string]interface{}
 	}{
+		{input: "{{url_encode('test}aaa')}}", expected: "test%7Daaa", extra: map[string]interface{}{}},
```

Comment:
The following two scenarios might also worth adding:

```go
{input: "{{}}", expected: "{{}}", extra: map[string]interface{}{}},
{input: `"{{hex_encode('PING')}}"`, expected: `"50494e47"`, extra: map[string]interface{}{}},
```

Reply:
done